requirements.in

```diff
@@ -1,5 +1,4 @@
 aws-cdk-lib
-aws-cdk.aws-lambda-python-alpha
 boto3
-requests
 pycryptodome
+pydantic
```

Comment:
What's this?

Reply:
Provides very easy and in this case forgiving json parse to object functions.